infrastructure/cdn-in-a-box/traffic_portal/Dockerfile

```diff
@@ -41,7 +41,6 @@ ARG TO_HOST=$TO_HOST
 RUN dnf install -y epel-release && \
     dnf install -y \
       jq \
-      nodejs \
```

Comment:
You need [this line](https://github.com/nodesource/distributions#using-ubuntu-5). The Enterprise Linux section doesn't include the install nodejs part, but if you look at the `setup_XX.x` scripts they tell you to install nodejs afterwards. This is why the CiaB is failing with unable to find `npm`.

Reply:
I figured, but I wanted to find out for myself. Letting GHA try to do it is perhaps not the _fastest_ way to find out, but it's by far the _easiest_.